deploy/jenkins-ci/Makefile

```diff
@@ -106,18 +126,19 @@ ifdef DOCKER_PASSWORD
 	@docker login -u "$(DOCKER_USER)" -p "$(DOCKER_PASSWORD)" docker.io
 endif
 endif
-	DOCKER_VERSION="v$(BACKEND_VERSION)" \
+	DOCKER_VERSION="$(DOCKER_TAG)" \
 	  CONSOLE_VERSION="$(CONSOLE_VERSION)" \
 	  CONSOLE_LOCAL_DIR="$(CONSOLE_LOCAL_DIR)" \
 	  make docker-build docker-push
 ifneq ($(IS_SNAPSHOT),y)
-	docker tag "$(DOCKER_NAME):v$(BACKEND_VERSION)" "$(DOCKER_NAME):v$(BACKEND_VERSION_BRANCH)"
+	docker tag "$(DOCKER_NAME):$(DOCKER_TAG)" "$(DOCKER_NAME):v$(BACKEND_VERSION_BRANCH)"
 	docker push "$(DOCKER_NAME):v$(BACKEND_VERSION_BRANCH)"
 	docker rmi "$(DOCKER_NAME):v$(BACKEND_VERSION_BRANCH)"
 endif
 	docker rmi "$(DOCKER_NAME):v$(BACKEND_VERSION)"
```

Comment:
It fails here... You missed to replace the `v$(BACKEND_VERSION)`

Reply:
Thanks for the heads up

I wonder why it didn't fail on my env... I suppose i need a way to clean it before a test.